LINE_SDK_Unity/Assets/LineSDK/LineSDK.cs

```diff
@@ -42,9 +42,13 @@ public class LineSDK: MonoBehaviour {
                 instance = this;
             } else if (instance != this) {
                 Destroy(gameObject);
+                return;
             }
             DontDestroyOnLoad(gameObject);
-            SetupSDK();
+
+            if (!string.IsNullOrEmpty(channelID)) { 
+                SetupSDK();
+            }
```

Comment:
```suggestion
            } else {
                Debug.LogWarning("LINE SDK channel ID is not set.");
            }
```

Reply:
I considered to add an `else` here but decided not to do that finally. If we treating "leaving empty and calling Setup manually" as a possible and normal case, we should not produce a warning for this case.

Instead, an SDK error and/or the exception in `Setup` method will play the role of telling something goes wrong.